data/brands/office/consulting.json

```
@@ -41,6 +41,16 @@
         "office": "consulting"
       }
     },
+    {
+      "displayName": "Ecovis International",
+      "locationSet": {"include": ["001"]},
```

It looks like they only have locations in Europe.
```suggestion
      "locationSet": {"include": ["150", "tr"]},
```